packages/typography/src/Typography.story.tsx

```diff
@@ -24,6 +33,23 @@ storiesOf('Typography', module).add('Default', () => (
       <InlineCode className={displayBlock}>InlineCode</InlineCode>
       <Disclaimer className={displayBlock}>Disclaimer</Disclaimer>
       <Overline>Overline</Overline>
+      <Link
+        href={select(
+          'href',
```

Comment:
We should indicate which typography element each of these knobs apply to. Right now, it's hard to tell which element you're changing props for.

Even better, you could make them each their own page under Typography to remove any confusion.

Reply:
Thought it was nice to see everything all together... would it be alright if I leave it this way but make the knobs more clear?

Reply:
Sure!

Reply:
Sure!